### ISSUE TITLE
Chicken overcome fence gate

### ISSUE THREAD

**Issue by JustForVS** [anegostudios/VintageStory-Issues#2231]
Game Version: 1.17.6
Platform: Windows 10
Modded: Yes (hudclock-2.8.0, ProspectorInfo-4.1.1, singlepause-1.0.3, ZoomButton1.3.0)
SP/MP: Singleplayer

### Description
Chicken overcome a fence gate two times this winter. I have two gates right behind each other to enter without chickens flee and a chicken was catched two times between these gates (screenshot). There was a high snow layer in front of the inner gate.
I build the second level of fences after the first time, so not sure if it is a specific problem with the gates or if the snow+fence issue is still not completely solved.

### How to reproduce
not sure how to force this

### Expected behavior
If not obvious, what did you expect to happen?

### Screenshots
![2022-10-05_21-15-58](https://user-images.githubusercontent.com/87942616/194157888-10343c70-9a80-435e-942e-e862c3b51624.png)

### Logs
```
Paste crash between the triple backticks.

In the game's Logs folder you may be able to find more information
inside client-main.txt and server-main.txt. Alternatively, you may use
gist.github.com to upload logs, linking them here, if they're too large.
```

**Comment by JustForVS** [comment]
happened again, no snow in front of the gate (not sure, if I removed it).
![2022-10-06_10-14-51](https://user-images.githubusercontent.com/87942616/194259306-16f0bafa-5706-491d-853b-4fe4cd2f7a65.png)

**Comment by JustForVS** [comment]
Okay, it's not the snow.

**Comment by Craluminum2413** [comment]
Closed due to inactivity, may also be fixed in 1.20 due to better physics
